test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -16,9 +16,9 @@
 @pytest.mark.integration("smdebug")
 @pytest.mark.parametrize("ec2_instance_type", SMDEBUG_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
-    # p2.8xlarge and m4.16xlarge TF1 Pipeline Test are failing for unknown reason.
+    # TF1 Pipeline Test are failing for unknown reason. Especially on p2.8xlarge and m4.16xlarge
```

Comment:
Nit: can you update the comment to be `TODO: Remove this once test timeout has been debugged (failures especially on p2.8xlarge)`

Reply:
Updated